arches_controlled_lists/src/arches_controlled_lists/components/editor/AddValue.vue

```diff
@@ -103,9 +103,8 @@ const addValue = () => {
 <template>
     <Button
         class="add-value"
-        raised
         icon="fa fa-plus-circle"
-        :severity="shouldUseContrast() ? CONTRAST : PRIMARY"
+        :severity="shouldUseContrast() ? CONTRAST : SECONDARY"
```

Comment:
I think you'll need to remove the `PRIMARY` import and add the `SECONDARY` import here to keep typescript happy.